packages/mock-block-dock/README.md

```diff
@@ -36,6 +36,13 @@ It will also pass `linkGroups` and `linkedEntities`, which will be populated onc
 
 `MockBlockDock` is automatically included in [block-template](https://www.npmjs.com/package/block-template), which you can copy via [create-block-app](https://www.npmjs.com/package/create-block-app)
 
+## Updating Props
+
+Any props passed to the child of `MockBlockDock` are automatically updated whenever:
+
+1.  The list of entities passed to the `mockDataStore` change.
+1.  The entityId passed to the child changes.
+
```

Comment:
We already mention updating props above so I've suggested adding this note there rather than introducing a new section talking about updating props (which repeats some of what is above).
```suggestion

The block will also be re-rendered with new properties if you update them on the child directly (e.g. if you are supplying the block component wrapped by `MockBlockDock` with props from some outside state).

`MockBlockDock` is automatically included in [block-template](https://www.npmjs.com/package/block-template), which you can copy via [create-block-app](https://www.npmjs.com/package/create-block-app)

```